exporters/otlp-http/metrics/src/main/java/io/opentelemetry/exporter/otlp/http/metrics/OtlpHttpMetricExporterBuilder.java

```diff
@@ -117,6 +117,11 @@ public OtlpHttpMetricExporterBuilder setAggregationTemporality(
     return this;
   }
 
+  OtlpHttpMetricExporterBuilder exportAsJson() {
```

Comment:
not exposing as a public API. Keeping the one on OkHttpMetricExporterBuilder public though for testing purposes.